app/_src/policies/meshmetric.md

```diff
@@ -231,6 +238,29 @@ scrape_configs:
         client_id: "prometheus-two"
 ```
 
+#### OpenTelemetry (experimental)
+
+```yaml
+backends:
+  - type: OpenTelemetry
+    openTelemetry: 
+      endpoint: http://otel-collector.observability.svc:4317
+```
+
+This configuration tells {{site.mesh_product_name}} Dataplane Proxy to push metrics to [OpenTelemetry collector](https://opentelemetry.io/docs/collector/).
+Dataplane Proxy will scrape metrics from Envoy and other [applications](/docs/{{ page.version }}/policies/meshmetric/#applications) 
+and push them to configured OpenTelemetry collector.
```

Comment:
nit: and other applications in a Pod/VM.

to emphasise that it's not random applications in kube clusters

Reply:
changed